<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>155</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>50</y>
     <width>360</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="noBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="info">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>10</y>
     <width>360</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Menlo</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(253, 105, 80)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
